apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/utils/getOrderParams.ts

```diff
@@ -38,18 +38,20 @@ export function getOrderParams(
   const balance = balances[order.inputToken.address]?.value
   const allowance = allowances[order.inputToken.address]?.value
 
-  let hasEnoughBalance, hasEnoughAllowance
-
-  if (order.partiallyFillable) {
-    // When balance or allowance are undefined (loading state), show as true
-    // When loaded, check there's at least PERCENTAGE_FOR_PARTIAL_FILLS of balance/allowance to consider it as enough
-    const amount = sellAmount.multiply(PERCENTAGE_FOR_PARTIAL_FILLS)
-    hasEnoughBalance = balance === undefined || isEnoughAmount(amount, balance)
-    hasEnoughAllowance = allowance === undefined || isEnoughAmount(amount, allowance)
-  } else {
-    hasEnoughBalance = isEnoughAmount(sellAmount, balance)
-    hasEnoughAllowance = isEnoughAmount(sellAmount, allowance)
-  }
+  const [hasEnoughBalance, hasEnoughAllowance] = (() => {
+    if (order.partiallyFillable) {
+      // When balance or allowance are undefined (loading state), show as true
```

Comment:
This comment is no longer true as it'll be undefined instead of `true`

Reply:
Fixed, also small refactor of the 2 branches